### PR TITLE
Fix: Fixing skipped test TestFileDownloadTokenMovement

### PR DESCRIPTION
**Pull request by nlok5923**
PR ready for review.

### CODE REVIEW

**Comment by nlok5923** [comment]
TestFileDownloadTokenMovement test are passing in CI
image
But overall CI failing due to file test
image

File: tests/cli_tests/zboxcli_download_token_test.go [hunk 1]
```diff
@@ -18,62 +19,47 @@ func TestFileDownloadTokenMovement(t *testing.T) {
 	t.Parallel()
 
 	t.Run("Each blobber's read pool balance should reduce by download cost", func(t *testing.T) {
```
Comment (Kishan-Dhakan) [inline tests/cli_tests/zboxcli_download_token_test.go:21]: Change the description to say "Downloader's readpool balance should reduce by download cost", current description applies to an older logic which is no longer applicable

**Review by Kishan-Dhakan — CHANGES_REQUESTED** [review]
Minor, LGTM

File: tests/cli_tests/zboxcli_download_token_test.go (hunk 1 above)
Comment (nlok5923) [reply tests/cli_tests/zboxcli_download_token_test.go]: Fixed at latest commit.

**Review by Kishan-Dhakan — APPROVED** [review]
LGTM

**Comment by nlok5923** [comment]
Download file token movement test are passing 
image
But overall CI failing due to these tests which are unrelated to this pr 
image

**Pull request merged**